components/arm/fake/zero_model.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "fake_arm",
+    "links": [
+        {
+            "id": "base_link",
+            "parent": "world",
+            "translation": {
+                "x": 500.0,
+                "y": 0,
+                "z": 300.0
+            },
+            "orientation": {
+                "type": "ov_degrees",
+                "value": {
+                    "x": 0.0,
+                    "y": 0.0,
+                    "z": 1.0,
+                    "th": 0.0
+                }
+            }
+        }
+    ],
+    "joints": []
+}
```

Comment:
```suggestion

}
```

Reply:
is this supposed to be an extra line before or after the close bracket?

Reply:
nice catch, after.